src/components/dashboard/main-content/course-enrollments/course-cards/InProgressCourseCard.jsx

```diff
@@ -196,7 +195,6 @@ export const InProgressCourseCard = ({
     updateCourseEnrollmentStatus({
       courseRunId: response.courseRunId,
       newStatus: response.courseRunStatus,
-      savedForLater: response.savedForLater,
```

Comment:
Good catch to figure out this line was not really needed (i.e., not `savedForLater` field in the serialized enrollment).